src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -618,11 +652,29 @@ export default class AircraftController {
             icao: aircraftTypeDefinition.icao,
             model: aircraftTypeDefinition,
             routeString: spawnPatternModel.routeString,
+            commands: spawnPatternModel.commands,
             // TODO: this may not be needed anymore
             waypoints: _get(spawnPatternModel, 'waypoints', [])
         };
     }
 
+    /**
+     * Execute a command on a new or preSpawned aircraft, ignoring controllability flag
+     *
+     * @param aircraft {AircraftModel}
+     * @param command {String}
+     * @private
+     */
+    _runCommandOnNewAircraft(aircraft, commandString) {
+        const command = new CommandParser(`${aircraft.getCallsign()} ${commandString}`).parse();
+        const defaultControllable = aircraft.isControllable;
+        aircraft.isControllable = true;
+
+        this._aircraftCommander.runCommands(aircraft, command.args, true);
+
+        aircraft.isControllable = defaultControllable;
```

Comment:
This just forces all aircraft to be controllable, which we don't want. If we did, we'd do it a different way. Should be handled within `AircraftCommander.runCommands()` as I described in a previous comment, and removed here.

Reply:
done